controllers/constant/odlm.go

```diff
@@ -845,6 +845,11 @@ spec:
                 memory: 512Mi
             logLevel: info
             primaryUpdateStrategy: unsupervised
+            primaryUpdateMethod: switchover
+            enableSuperuserAccess: true
+            replicationSlots:
+              highAvailability:
+                enabled: false
```

Comment:
`enabled: false` should this be updated depending on the profile (1 replica vs multiple replicas)?

Reply:
resolved via slack - the values simply ensure 1.18 vs 1.22 default values compatibility